code/week3/week3.cpp

```diff
@@ -0,0 +1,274 @@
+#include "week3.h"
+
+int main() {
+     using namespace std;
+
+     // ******************** ARRAYS *********************
+     // array docs http://en.cppreference.com/w/cpp/container/array
+
+     // initialization
+     // array<TYPE, SIZE> name = {elements}
+     array<int, 3> int_array = {1, 2, 3};
+     array<string, 3> string_array = {"arrays", "are", "cool"};
+
+     // You can get the number fo elemnts in std::array by using .size()
+     cout << "size of the array is = " << int_array.size() << endl;
+
+     // remember arrays index 0,1,2,...
+     cout << "printing int_array" << endl;
+     for(int i = 0; i < int_array.size(); i++) {
+         // .at(index) returns the element at that index with bounds checking at runtime
+         cout << "int_array at index = " << i << " the value is = " << string_array.at(i) << endl;
```

Comment:
you are indexing into string_array, but the printout says you are indexing into int_array